weave/trace_server/calls_query_builder/optimization_builder.py

```diff
@@ -423,12 +423,7 @@ def _create_like_optimized_eq_condition(
         # Empty string is not a valid value for LIKE optimization
         return None
 
-    # Boolean literals are not wrapped in quotes in JSON payloads
-    if literal_value in ("true", "false"):
-        like_pattern = f"%{literal_value}%"
-    else:
-        like_pattern = f'%"{literal_value}"%'
```

Comment:
This was added to support filtering on booleans. No longer necessary?

Reply:
previously we had quotes, which is an extra optimization. now we have no quotes, which should work for both normal strings and bools. slightly less optimized but whatever